binding.gyp

```diff
@@ -8,14 +8,14 @@
     ['AR', '<(module_root_dir)/mason_packages/.link/bin/llvm-ar'],
     ['NM', '<(module_root_dir)/mason_packages/.link/bin/llvm-nm']
   ],
-  'includes': [ 'common.gypi' ], # brings in a default set of options that are inherited from gyp
+  'includes': [ 'common.gypi'], # brings in a default set of options that are inherited from gyp
   'variables': { # custom variables we use specific to this file
       'error_on_warnings%':'true', # can be overriden by a command line variable because of the % sign using "WERROR" (defined in Makefile)
-      # Use this variable to silence warnings from mason dependencies and from NAN
+      # Use this variable to silence warnings from mason dependencies
       # It's a variable to make easy to pass to
       # cflags (linux) and xcode (mac)
       'system_includes': [
-        "-isystem <(module_root_dir)/<!(node -e \"require('nan')\")",
+        "-isystem <!@(node -p \"require('node-addon-api').include.slice(1,-1)\")",
```

Comment:
@artemp what's the need for slicing the include instead of using 

```
"<!@(node -p \"require('node-addon-api').include\")"
```

?

Reply:
@mapsam this is the only way I found that works for me. The above expansion has too many quotes. Pls, suggest workaround, it's probably lack JS knowledge on my behalf.